### PR TITLE
Releasing 1.0.0

### PR DESCRIPTION
**Pull request by jamesagardner**

### CODE REVIEW

**Comment by jamesagardner** [comment]
I think the above failure is pointing out that the master branch still has the old Travis CI distribution, which is no longer working with Oracle JDK 8.  I believe the commit to use the trusty distribution fixes the issue (`travis-ci/pr` passed, but `travis-ci/pull` failed).

So I think we're OK to continue with the merge to master assuming the code review passes.  I'm hoping to finish this PR on Monday and release 1.0.0 at that time.

File: marklogic-junit5/examples/simple-dhf4/build.gradle [hunk 1]
```diff
@@ -10,9 +10,9 @@ repositories {
 }
 
 dependencies {
-  mlRestApi "com.marklogic:marklogic-unit-test-modules:0.13.develop"
+  mlRestApi "com.marklogic:marklogic-unit-test-modules:1.0.beta"
 
-	testCompile "com.marklogic:marklogic-junit:0.13.develop"
+	testCompile "com.marklogic:marklogic-junit5:1.0.beta"
```
Comment (hansenmc) [inline marklogic-junit5/examples/simple-dhf4/build.gradle]: Do not publish with a dependency on a beta release. Need to update junit5 and marklogic-unit-test-modules dependencies

File: marklogic-junit5/examples/simple-ml-gradle/README.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ Next, add the following dependencies - this assumes you're using Gradle version
     dependencies {
       // existing dependencies
       
-      testCompile "com.marklogic:marklogic-junit:0.10.0"
+      testCompile "com.marklogic:marklogic-junit5:1.0.beta"
```
Comment (hansenmc) [inline marklogic-junit5/examples/simple-ml-gradle/README.md]: Same issue. Shouldn't publish with dependency on a beta release, use a final release

File: marklogic-junit5/examples/simple-ml-gradle/build.gradle [hunk 3]
```diff
@@ -15,11 +15,11 @@ repositories {
 }
 
 dependencies {
-  mlBundle "com.marklogic:marklogic-unit-test-modules:0.13.develop"
+  mlBundle "com.marklogic:marklogic-unit-test-modules:1.0.beta"
 
 	compile "com.marklogic:marklogic-client-api:4.2.0"
 
-	testCompile "com.marklogic:marklogic-junit:0.13.develop"
+	testCompile "com.marklogic:marklogic-junit:1.0.beta"
```
Comment (hansenmc) [inline marklogic-junit5/examples/simple-ml-gradle/build.gradle]: don't publish with beta dependencies

**Comment by hansenmc** [comment]
I noticed that the marklogic-unit-test-modules test-helper.xqy references "`ROXY-BOOM`":

https://github.com/marklogic-community/marklogic-unit-test/blob/develop/marklogic-unit-test-modules/src/main/ml-modules/root/test/test-helper.xqy#L50

Not sure if we want to leave that in there as an ode to it's origins, or if we wanted to remove all references to Roxy.

**Comment by hansenmc** [comment]
Also, before publishing the copyright year should be updated to the current year. There are several files that have `copyright (c) 2018 MarkLogic Corporation`

File: marklogic-junit5/examples/simple-ml-gradle/build.gradle (hunk 3 above)
Comment (jamesagardner) [reply marklogic-junit5/examples/simple-ml-gradle/build.gradle]: fixed in pr #101

File: marklogic-junit5/examples/simple-ml-gradle/README.md (hunk 2 above)
Comment (jamesagardner) [reply marklogic-junit5/examples/simple-ml-gradle/README.md]: Fixed in Pr #101

File: marklogic-junit5/examples/simple-dhf4/build.gradle (hunk 1 above)
Comment (jamesagardner) [reply marklogic-junit5/examples/simple-dhf4/build.gradle]: Fixed in pr #101

**Pull request merged**